examples/fungible/src/lib.rs

```diff
@@ -242,6 +243,8 @@ pub enum ApplicationCall {
         amount: Amount,
         target_account: Account,
     },
+    /// A request for this fungible token's ticker symbol.
+    TickerSymbol,
```

Comment:
Maybe we should design a generic way to read access the parameters of a known application? (if that's not the case yet)